packages/dcos-integration-test/extra/test_dcos_checks.py

```diff
@@ -0,0 +1,25 @@
+import subprocess
+
+
+def invoke_check_binary(target, *args):
+    """
+    Run the dcos-checks binary.
+
+    :param target: dcos-checks subcommand
+    :param args: a list of target arguments passed to a dcos-checks binary
+    :return: combined stdout/stderr, return code
+    """
+    try:
+        output = subprocess.check_output(
+            ["/opt/mesosphere/bin/dcos-checks", "--role", "master", target] + [arg for arg in args],
```

Comment:
For quite a while now its been a requirement that all of these tests are run from a master. I don't plan on this changing any time soon so I do not find this case to be reliant on happenstance. Really, I would like us to have the integration tests have ZERO shelling out, but for now this is an acceptable evil

I do feel that it is less than ideal to add a test that makes assumptions about the current location of the test file rather than only assuming there is a cluster. So, perhaps add a TODO/FIXME/JIRA that says "make this test runnable from any node" or something?

As for SSH facility in tests, we are getting closer but I do not think this case is worth jumping the gun and adding more SSH dependency before its ready